src/main/java/quotify_app/entities/PredictionRequest.java

```diff
@@ -0,0 +1,133 @@
+package quotify_app.entities;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Class that specifies the format a request should take for the prediction model.
+ */
+public final class PredictionRequest {
+    private final int beds;
+    private final int baths;
+    private final double universalsize;
```

Comment:
some of these variables dont follow the standard java naming convention, e.g universalsize => universalSize.
this is an easy refactor just change the name.